### ISSUE TITLE
Accessibility Feedback for Staging Review [Authenticated Experience - Direct Deposit for Edu]

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#19495]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

VSA: Josh
VSP: Trevor

## Details

This epic contains the accessibility feedback for Staging Review - Authenticated Experience - Direct Deposit for Edu.

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Access Information

Access information for the tool or feature in staging _**do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository**_

* Staging url: https://staging.va.gov/profile

* Test cases (itemized steps to test): [VSA TestRail test cases](https://dsvavsp.testrail.io/index.php?/suites/view/5&group_by=cases:section_id&group_order=asc&group_id=402)

<details><summary>tasklist for github project tracking</summary>

```[tasklist]
### Tasks
- [ ] #65506
``` 

</details>

**Comment by joshkimux** [comment]
<img width="432" alt="Screen Shot 2021-02-05 at 1 22 58 PM" src="https://user-images.githubusercontent.com/14154792/107073441-61a6e080-67b5-11eb-9d51-6b10e9b48cb4.png">

@sporeboy Would it be possible to use radio buttons instead of a dropdown here? For sighted users, options will be more immediate. For screen reader users, additional context is naturally provided e.g. "Savings, 1 of 2, selected" without the "check mark" empty selection.

<img width="1431" alt="Screen Shot 2021-02-05 at 1 26 22 PM" src="https://user-images.githubusercontent.com/14154792/107073749-bea29680-67b5-11eb-857f-82558b8906a1.png">

Technically this still meets compliance standards, so it's not of concern if it's too late to fix. The UX benefits for sighted and screen reader users would be really nice though!

---

A fun video on this topic 🤓 
[F*** dropdowns - SXSW Keynote](https://www.fuckdropdowns.com/)

**Comment by joshkimux** [comment]
@jason-gcio @erikphansen I'll be logging accessibility tickets for this staging review in this epic! If either of you have questions or think someone else should be tagged on this, do let me know 😄 

I'll aim to get this completed by the end of this week 🙇

**Comment by erikphansen** [comment]
Thanks @joshkimux. Might want to keep @mshea0606 in the loop as well. This feature is weird as the work has spanned both the Authenticated Experience and the E-Benefits teams.

**Comment by joshkimux** [comment]
@sporeboy This isn't accessibility compliance feedback, but something to possibly consider! Could we explore formatting the bank fraud notice as a heading as opposed to just a note alert? This would make it significantly more discoverable for all users-- but specifically screen reader users as it will appear in the page's natural headings list.

<img width="1465" alt="Screen Shot 2021-02-08 at 6 21 49 PM" src="https://user-images.githubusercontent.com/14154792/107294122-d9d20800-6a3a-11eb-9153-5ce30fc02bf3.png">

**Comment by joshkimux** [comment]
<img width="1020" alt="Screen Shot 2021-02-08 at 6 25 31 PM" src="https://user-images.githubusercontent.com/14154792/107294308-2e758300-6a3b-11eb-89e9-9d327880754f.png">

For more context, the current location of the note makes it appear to be only for the first `h3`, when I believe it is meant to apply for all the sections instead.

**Comment by joshkimux** [comment]
@1Copenut I've completed my accessibility feedback. This is ready for your review! ✌️

**Comment by joshkimux** [comment]
@mshea0606 @jason-gcio @erikphansen 
The accessibility review is complete 🥳 
VSP will be reviewing this come next Monday-- but in the meantime I wanted to raise awareness of the 3 launchblocker tickets on this epic.
- [Form field instructions MUST be associated with inputs and buttons](https://github.com/department-of-veterans-affairs/va.gov-team/issues/19570)
- [Form fields and their unique IDs MUST NOT be duplicated on the page](https://github.com/department-of-veterans-affairs/va.gov-team/issues/19594)
- [An equivalent alternative MUST be provided for the image of the check](https://github.com/department-of-veterans-affairs/va.gov-team/issues/19611)

Luckily, the image of check launchblocker is an easy fix! The content team has already provided a fix, so we just need sign off from the designer @sporeboy and y'all for it to be good to go. 🎉 

The other two may be a bit more tricky, and I'm more than happy to hop on a call to help y'all find the best path forwards to get them resolved 🙇

**Comment by joshkimux** [comment]
@andaleliz I love this proposed solution! Fingers crossed this can be made into a reality 🤞

**Comment by andaleliz** [comment]
@joshkimux thank you for looking at it! I _just_ saw the mock-up you posted where you had it at the bottom of the page too! Our PM only put the other one in my ticket 😆 great minds.

Glad we're on the same page. Our team will make a FE ticket for this and it WILL get made into a reality! Also, we're doing the `select` > `radio button` change you suggested too.  I wish I had seen both of these sooner. Thanks for pointing these things out.

@mtcA6 I guess we need to keep this epic open since there's till one ticket assigned to Adam in the icebox.

**Comment by mtcA6** [comment]
@andaleliz sorry I didn't copy both items over to the ticket! Glad you arrived at the same conclusion/ saw the second one. 
Adam is working on the last ticket in this epic this sprint and I was going to close that out and prioritize the work you just completed from #63493 and #63495 and work through those separately.
